### PR TITLE
Made sure there exists a select child and has values

### PR DESCRIPTION
**Pull request by makian123**

### CODE REVIEW

File: src/cogs/modmail.py [hunk 1]
```diff
@@ -60,11 +60,12 @@ async def select_type(self, interaction, selection):
 
     @discord.ui.button(label = "Create ticket", custom_id = "persistent_view:create_ticket")
     async def button_callback(self, interaction, button):
-        if not self.children[0].values:
-            await interaction.response.send_message("Please select an option", ephemeral = True)
-            return
+        for child in self.children:
```
Comment (Tsche) [inline src/cogs/modmail.py:63]: can there be more than one child?

Comment (makian123) [reply src/cogs/modmail.py:63]: Yes, the button can be too, im thinking maybe something gets messed up and button is the 0th child

Comment (Tsche) [reply src/cogs/modmail.py:63]: gotcha

**Review by Tsche — APPROVED** [review]
lgtm

**Review by Yuhanun — APPROVED** [review]

**Pull request merged**